src/Microsoft.Extensions.Logging.Abstractions/ILoggerFactory.cs

```diff
@@ -19,9 +19,13 @@ public interface ILoggerFactory : IDisposable
         ILogger CreateLogger(string categoryName);
 
         /// <summary>
+        /// <para>
+        /// This method is obsolete, use AddProvider on the concrete Microsoft.Extensions.Logging.LoggerFactory class.
```

Comment:
Can we follow the guidelines on Recommend following a pattern closer to the guidelines: https://github.com/aspnet/Home/wiki/Engineering-guidelines#breaking-changes a little more closely? I recommend using the exact phrasing in there, i.e. `This method is obsolete and will be removed in a future version. The recommended alternative is Microsoft.Extensions.Logging.LoggerFactory.AddProvider().`